plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -22,6 +23,8 @@
 author:
   - Werner Dijkerman (@dj-wasabi)
   - Guillaume Martinez (@Lunik)
+  - Lennert Mertens (@LennertMertens)
+  - Stef Graces (@stgrace)
```

Comment:
How is @stgrace related to this PR?

Also please note that adding both names to the author list means that you sign up as maintainers for this module.

Reply:
I helped my colleague Lennert with this feature, and I am willing to be a maintainer.
We started off using it in a private repo and to contribute it back to here the git history got altered a little bit